.ahub/tcchecker-tca/config.yaml

```diff
@@ -0,0 +1,43 @@
+version: 2
+test:
+  - name: NN Runtime
+    testCaseLanguage: CPP
+    testFW: GTEST
+    testCaseFolder:
+      - ./compute/test/cker
+      - ./runtime/onert/core/src/backend/cpu_common
+      - ./runtime/onert/frontend/nnapi
+      - ./runtime/onert/test/core/compiler
+      - ./runtime/onert/test/core/exec
+      - ./runtime/onert/test/core/interp
+      - ./runtime/onert/test/graph
+      - ./runtime/onert/test/graph/operand
+      - ./runtime/onert/test/graph/operation
+      - ./runtime/onert/test/graph/verifier
+      - ./runtime/onert/test/ir
+      - ./runtime/onert/test/util
+      - ./tests/nnapi/src
+      - ./tests/nnfw_api/src
+      - ./tests/tools/tflite_run/src
+
+    testFile:
+      - extension: cpp
+        any: true
+      - extension: cc
+        any: true
+
+    testCase:
+      - condition:
+        - functionName:
+            starts:
+              - TEST
+ 
+    negativeTestCase:
+      - condition:
+        - testName:
+            starts:
+              - neg_
```

Comment:
Please note that this rule does not apply NN API tests(`./tests/nnapi/src`). Would this be OK?

Reply:
@wateret , perhaps you are talking about the *.nnfw named test cases of Generated Test? Unfortunately, the TC is generated through the build and cannot be counted by TCM. TCM counts without a build.

On the other hand, the current TCM agent fails to properly parse the C++ code generated by the Generated Test and dies. Perhaps the supported C++ versions are different.

At the moment, I think it's okay to leave. :)

Reply:
I think the reason why it dies is that there is no test case code in C++. In our build system, all the test cases written in python are converted to C++ before building nnapi test. There is literally no test cases in C++ in the repo.